tests/Doctrine/Tests/KeyValueStore/Storage/MongoDbStorageTest.php

```diff
@@ -33,7 +33,10 @@ class MongoDbStorageTest extends \PHPUnit_Framework_TestCase
 {
     protected function setUp()
     {
-        $this->mongo = $this->getMock('\Mongo');
+        $this->mongo = $this
+            ->getMockBuilder('\Mongo')
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
I think you could achieve the same with `$this->createMock('\Mongo');

Reply:
It’s not supported by all the phpunit versions currently allowed in require-dev, I prefer to  update them in another PR with the phpunit version